tests/code_size/hello_webgl2_fastcomp_wasm.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 563,
   "a.html.gz": 377,
-  "a.js": 5288,
-  "a.js.gz": 2535,
-  "a.wasm": 8004,
-  "a.wasm.gz": 4376,
-  "total": 13860,
-  "total_gz": 7288
+  "a.js": 5300,
+  "a.js.gz": 2541,
+  "a.wasm": 5155,
+  "a.wasm.gz": 2719,
+  "total": 11006,
+  "total_gz": 5637
```

Comment:
This is a nice size win, but if it's from using more `Math.*` then it may cause a slowdown. We've hand-picked some math functions over time to be in C and others not, to make tradeoffs in this space. I don't think it's worth making a big change to that right before removing fastcomp.

For upstream, though, I agree we should not be using these functions in library.js.

Reply:
Also, if this PR increases the difference between upstream and fastcomp, that has downsides too (like people comparing on the last fastcomp version would see it has better code size but slower speed than upstream, confusingly).